docs/dietpi_references.md

```diff
@@ -0,0 +1,23 @@
+# References
+
+At this page, links to general information are referenced.  
+Links to special software packages are located at the end of the according software package description in the "Optimized Software" documentation area.
+
+## General links
+
+- Twitter: <https://twitter.com/dietpi_>
```

Comment:
Let's use `https://twitter.com/DietPi_` here, it practically makes no difference but matches what you get when opening the profile page from within Twitter.

Reply:
Ok, changed.